src/components/principal/Principal.vue

```diff
@@ -12,6 +12,7 @@
       </div>
     </div>
     <ConfirmationDialog v-if="displayConfirmationDialog" :unit="activeUnit"></ConfirmationDialog>
+    <ResourceDialog v-if="displayViewResourceDialog" :resource="activeResource" :unit="activeUnit"></ResourceDialog>
     <AddResourceForm v-if="displayResourceForm" ></AddResourceForm>
```

Comment:
do we still need this? I don't think so, since resource dialog will be used instead